src/subcommand.rs

```diff
@@ -84,6 +84,13 @@ impl Subcommand {
             package.as_ref().map(|s| &**s),
         )?;
         let root_dir = manifest.parent().unwrap();
+
+        let target_dir = target_dir.or_else(|| {
+            std::env::var_os("CARGO_BUILD_TARGET_DIR")
+                .or_else(|| std::env::var_os("CARGO_TARGET_DIR"))
```

Comment:
Not sure if `--manifest-path` or the environment variable should get precedence.